test/parallel/test-fs-rmSync-special-char.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fs = require('fs');
```

Comment:
```suggestion
require('../common');
```

All the test needs to load the fixtures, see  https://github.com/nodejs/node/blob/260092eca316bae6036e52b2cdc73c4c4f8a9da6/doc/contributing/writing-tests.md#L58

Reply:
Updated. Sorry for the inconvenience. @joyeecheung